app/experimenter/experiments/tasks.py

```diff
@@ -122,100 +121,124 @@ def update_experiment_bug_task(user_id, experiment_id):
 def update_experiment_info():
     metrics.incr("update_experiment_info.started")
     logger.info("Updating experiment info")
-    launch_experiments = Experiment.objects.filter(
-        Q(status=Experiment.STATUS_ACCEPTED) | Q(status=Experiment.STATUS_LIVE)
-    )
+    update_accepted_experiments()
```

Comment:
I think if you fold these two back together you can just flatten them back into this function